pkg/ddevapp/laravel.go

```diff
@@ -78,3 +88,10 @@ func laravelPostStartAction(app *DdevApp) error {
 
 	return nil
 }
+
+// hasDdevPrefixedHost checks if the .env file contains a DB_HOST with ddev- prefix
+func hasDdevPrefixedHost(envFilePath string) bool {
+	envMap, _, _ := ReadProjectEnvFile(envFilePath)
+	existingDBHost := envMap["DB_HOST"]
+	return existingDBHost != "" && strings.HasPrefix(existingDBHost, "ddev-")
+}
```

Comment:
It works, but I'm not sure if `hasDdevPrefixedHost` is worth it, since it's unexpected, undocumented, and not really tested.

If we add it, there will be no way back without breaking changes, because people will start depending on it. We should consider it carefully.

If you want this PR merged in the next release cycle (~2 weeks), let's remove `hasDdevPrefixedHost` from here, keep only the change for `--omit-containers=db`, and discuss whether we want something like this in the issue queue.